src/utils/shorten-url.js

```diff
@@ -4,65 +4,66 @@
 
 // @flow
 
-import queryString from 'query-string';
-import url from 'url';
-
 const accessToken = 'b177b00a130faf3ecda6960e8b59fde73e902422';
-export function shortenUrl(urlToShorten: string): Promise<string> {
+export async function shortenUrl(urlToShorten: string): Promise<string> {
   let longUrl = urlToShorten;
   if (!longUrl.startsWith('https://profiler.firefox.com/')) {
-    const parsedUrl = url.parse(longUrl);
-    const parsedUrlOnCanonicalHost = Object.assign({}, parsedUrl, {
-      protocol: 'https:',
-      host: 'profiler.firefox.com',
-    });
-    longUrl = url.format(parsedUrlOnCanonicalHost);
+    const parsedUrl = new URL(longUrl);
```

Comment:
now node supports the native URL API so we don't need to use the module `url` anymore